src/app/shared/notification/service/notification.service.ts

```diff
@@ -45,4 +53,14 @@ export class NotificationService {
   public onFeatureNotAvailableYet() {
     this.addHintNotification('This feature is not available yet. But it is coming soon! :-)');
   }
+
+  public complexNavigatorAnnouncement() {
+    this.addComplexNavigatorAnnouncementNotification(
+      'Try out our new display: \n ' +
+      'the Complex Navigator!');
+  }
+
+  public closeAnnouncement(): void {
```

Comment:
As discussed, updated `addComplexNavigatorAnnouncementNotification` to return the result the toast.info call 
```
return this.toastrService.info(...);
```

Then, update `complexNavigatorAnnouncement` to also return
```
return this.addComplexNavigatorAnnouncementNotification(...)
```

Then, in complex-results.component.ts, store the returned value
```
this.toast = this.notificationService.complexNavigatorAnnouncement();
```

Update `closeAnnouncement` to take the toast id
```
  public closeAnnouncement(toastId: number): void {
    this.toastrService.clear(toastId);
  }
```

And finally, update the call to it
```
setComplexNavigatorView() {
    this.DisplayType = this.COMPLEX_NAVIGATOR_VIEW;
    this.reloadPage();
    if (!!this.toast) {
      this.notificationService.closeAnnouncement(this.toast.toastId);
      this.toast = null;
    }
  }
```